src/site/content/en/payments/how-payment-request-api-works/index.md

```diff
@@ -109,6 +109,15 @@ After setting the two parameters and creating the `request` object as shown
 above, you can call the `show()` method, which displays the payment app user
 interface.
 
+{% Aside 'warning' %}
+
+Starting from version 99, Chrome requires [a user
```

Comment:
Version 99 is incorrect; the deprecation was [extended to M100](https://groups.google.com/a/chromium.org/g/blink-dev/c/xCW746n6XJI/m/rpqi3Ws2EAAJ), and we may actually delay it further due to a partner who hasn't been able to adapt their code in time.

We could either rewrite this warning to be less specific about which milestone, or we can hold this PR until we are confident about which milestone we're going to ship in?

Reply:
I think it's important information that this function requires a user activation in the spec and on Chrome in the future. I've updated the phrase.

Reply:
LGTM, thanks!